Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md

```diff
@@ -6,10 +6,9 @@ As with all software, including releases, there is always a chance for issues an
 
 ---
 
-<h2 style="font-weight:bold" align="center">New 🎉</h2>
+<h2 style="font-weight:bold" align="center">New Features ✨</h2>
 
-1. Add ability to render rectangles to the scene with the various attributes below:  
-   GitHub Issues: [#75](https://github.com/KinsonDigital/Velaptor/issues/75)
+1. [#75](https://github.com/KinsonDigital/Velaptor/issues/75) - Add ability to render rectangles to the scene with the various attributes below:
```

Comment:
Refactor to match below:

```md
   - Solid/filled rectangles
   - Empty rectangles
   - Change the fill color of a filled rectangle
   - Change the border color of an empty rectangle
   - Add a horizontal color gradient with a start and end color
     - This can be applied to filled and empty rectangles
   - Add a vertical color gradient with a start and end color
     - This can be applied to filled and empty rectangles
   - Change the border thickness of an empty rectangle
   - Set the radius of each corner independently
     - This can be applied to filled and empty rectangles
```